metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache-2.0'
 description 'Installs/Configures tripwire_agent'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '0.1.8'
+version '0.1.9'
```

Comment:
We never actually released a 0.1.8, so please leave this.

Reply:
Oh interesting, I'll change it back.